components/Icon/Icon.module.css

```diff
@@ -27,3 +27,33 @@
     --size: 40px;
   }
 }
+
+.inline {
+  display: inline;
+  width: var(--size);
```

Comment:
The only difference in this style from the wrapper is `display: inline`. I think having the `.inline` class only include the `display: inline` is sufficient and just apply `styles.wrapper` at all times. then only conditionally apply `inline` if the prop is true. With the `classnames` library it'd look something like this

```js
    <IconSVG
      className={cn(styles.wrapper, styles[size], className, {
        inline: styles.inline,
      })}
    />
```

This means all the duplicated code will always be on, and only change the `display` if the prop is true. I believe order matters, so having inline at the bottom should always overwrite any previous styles above but that I'm not 100% sure about.

Reply:
Addressed (as you know thanks to our helpful pair-programming sesh) in https://github.com/gravitational/docs/pull/177/commits/13a8d198e70a716c62d643991c2dad28d3003a9b